Source/WebCore/editing/DictionaryPopupInfo.h

```diff
@@ -37,7 +38,7 @@ namespace WebCore {
 #if PLATFORM(COCOA)
 struct DictionaryPopupInfoCocoa {
     RetainPtr<NSDictionary> options;
-    RetainPtr<NSAttributedString> attributedString;
+    AttributedString attributedString;
```

Comment:
We should really see if we can get this down to just String (we should be able to, or at least nearly).

Reply:
Eventually, yes.  Changing it to an AttributedString is equally good for this purpose and less of a change in functionality.